src/services/activity-history-services/get-activity-history-total-records.js

```diff
@@ -0,0 +1,36 @@
+import { makeEventsListBaseUrl } from './make-events-list-base-url'
+import graphQLApi from '../axios/makeEventsApi'
+import { AxiosHttpClientAdapter } from '../axios/AxiosHttpClientAdapter'
+
+export const getActivityHistoryTotalRecords = async ({ search = '' }) => {
+  const apiClient = graphQLApi(import.meta.env.VITE_PERSONAL_TOKEN)
+  const offSetEnd = new Date()
+  const offSetStart = new Date(
+    Date.UTC(offSetEnd.getFullYear(), offSetEnd.getMonth(), offSetEnd.getDate() - 30)
+  )
+  const payload = {
+    operatioName: 'ActivityHistory',
+    query: `query ActivityHistory { activityHistoryEvents(aggregate: { count: rows }, filter: { tsRange: {begin:"${offSetStart.toISOString()}", end:"${offSetEnd.toISOString()}" } }) { count } } `
+  }
+
+  if (search) {
+    payload.query = `query ActivityHistory { activityHistoryEvents (aggregate: { count: rows }, filter: { tsRange: {begin:"${offSetStart.toISOString()}", end:"${offSetEnd.toISOString()}" } or: [{ titleIlike: "%${search}%" }, { commentIlike: "%${search}%" }], }) { count } } `
+  }
```

Comment:
```suggestion
  const payload = {
    operatioName: 'ActivityHistory',
    query: `query ActivityHistory { activityHistoryEvents(aggregate: { count: rows }, filter: { tsRange: {begin:"${offSetStart.toISOString()}", end:"${offSetEnd.toISOString()}" } }) { count } } `
  }

  if (search) {
    payload.query = `query ActivityHistory { activityHistoryEvents (aggregate: { count: rows }, filter: { tsRange: {begin:"${offSetStart.toISOString()}", end:"${offSetEnd.toISOString()}" } or: [{ titleIlike: "%${search}%" }, { commentIlike: "%${search}%" }], }) { count } } `
  }
```
Outro ponto é que devemos passar parâmetros pelo campo `variables`, similar a https://github.com/aziontech/azion-console-kit/blob/e43d0a1c17b3cc2e34b028e7258273da2c64e570/src/services/v2/billing-gql-service.js#L38